stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/attachments/AttachmentsPicker.kt

```diff
@@ -88,37 +62,16 @@ import java.io.File
  * @param onAttachmentsSelected Handler when attachments are selected and confirmed by the user.
  * @param onDismiss Handler when the user dismisses the UI.
  */
-@OptIn(ExperimentalPermissionsApi::class)
 @Composable
 public fun AttachmentsPicker(
     attachmentsPickerViewModel: AttachmentsPickerViewModel,
     onAttachmentsSelected: (List<Attachment>) -> Unit,
     onDismiss: () -> Unit,
     modifier: Modifier = Modifier,
+    tabFactories: List<AttachmentsPickerTabFactory> = AttachmentsPickerTabFactories.defaultFactories(),
```

Comment:
We have this as a param here, but not `ChatTheme` - What if our users want to disable e.g. Files and Media capture and only keep images?

We should expose this through the ChatTheme and have a composition local that allows the user to override it in the theme-level!